xml/cha_introduction.xml

```diff
@@ -296,4 +296,51 @@
   </important>
  </sect1>
 
+ <sect1 xml:id="sec-intro-support">
+  <title>Support</title>
+  <para>
+   Regardless of the public cloud you use and the plan you choose, &suse; has
+   you covered with support. Details however differ:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     PAYG instances include support through the cloud provider which in turn is
+     supported by &suse;. The CSP provides 1st and 2nd level, &suse; will
+     provide level 3 if necessary.
+    </para>
+    <para>
+     This means that you <emphasis>always contact your cloud service provider
+     first</emphasis>. They will assist you with your problem. If no resolution
+     can be foud, they will open a support ticket for you with &suse;.
+    </para>
```

Comment:
Similarly here we would just say, "Always contact your cloud service provider for assistance to your problem."

Reply:
Ditto this will also be incorrect in the not too distant future.